src/main/java/org/folio/circulation/domain/RequestRepository.java

```diff
@@ -26,7 +29,7 @@
   private final LoanRepository loanRepository;
   private final ServicePointRepository servicePointRepository;
   private final PatronGroupRepository patronGroupRepository;
-
+  private final Logger log;
```

Comment:
Is there a reason this is instantiated differently to the other loggers?

Reply:
Hm.. I'm pretty sure I copied this pattern from an existing class in the project.

Reply:
You probably did. It wouldn’t surprise me to find it is already inconsistent (I’m not used to static loggers, previous systems used to inject them)